packages/marketplace/src/resources/locales/es.json

```diff
@@ -0,0 +1,77 @@
+{
+  "translation": {
+    "settings": {
+      "title": "Ajustes",
+      "optionsHeading": "Opciones",
+      "starCountLabel": "Numero de estrellas",
+      "tagsLabel": "Etiquetas",
+      "devToolsLabel": "Herramientas para desarrolladores de temas",
+      "hideInstalledLabel": "Esconder instalado cuando buscando",
+      "colourShiftLabel": "Cambiar colores cada minuto",
+      "tabsHeading": "Pestañas",
+      "resetHeading": "Reajustar",
+      "resetBtn": "$t(settings.resetHeading)",
+      "resetDescription": "Borrar todas estensiones and temas, y borrar preferencias"
+    },
+    "tabs": {
+      "Extensions": "Extensiónes",
+      "Themes": "Temas",
+      "Snippets": "Codigos",
+      "Apps": "Aplicaciones",
+      "Installed": "Instalados"
+    },
+    "snippets": {
+      "addTitle": "Añadir Codigo",
+      "editTitle": "Editar Codigo",
+      "viewTitle": "Ver Codigo",
+      "customCSS": "Custom CSS",
+      "customCSSPlaceholder": "Crea tu propio CSS aqui! Puedes encontrarlos en la pestaña de instalados para administrarlos.",
+      "snippetName": "Nombre del codigo",
+      "snippetNamePlaceholder": "Asignale un nombre para tu codigo personalizado",
+      "snippetDesc": "Descripcion del codigo",
+      "snippetDescPlaceholder": "Crea una description para tu codigo personalizado",
+      "snippetPreview": "Codigo",
+      "optional": "Opcional",
+      "addImage": "Añadir imagen",
+      "changeImage": "Cambiar imagen",
+      "saveCSS": "Guardar CSS"
+    },
+    "reloadModal": {
+      "title": "Recargar",
+      "description": "Una recarga de ventada es necesaria para completar esta operación.",
+      "reloadNow": "Recargar ahora",
+      "reloadLater": "Recargar después"
+    },
+    "devTools": {
+      "title": "Herramientas de desarrollador de temas",
+      "noThemeInstalled": "Error: No tema de marketplace instalado",
+      "noThemeManifest": "Error: No manifiesto de tema encontrado",
+      "colorIniEditor": "Editor de Color.ini",
+      "colorIniEditorPlaceholder": "[your-colour-scheme-name]",
```

Comment:
This is a sample placeholder value written in English that can be translated.
You want to translate it or keep it in English?

Reply:
As long as it's not user-facing, it should be fine. However, if it is, I'll be happy to replace it to "[nombre-de-esquema-de-color]".

Reply:
It is user facing, it's what shows up in the text entry before you've typed anything.

Reply:
fixed